src/main/java/org/pentaho/mongo/wrapper/ConnectionStringMongoClientWrapper.java

```diff
@@ -0,0 +1,380 @@
+/*!
+ * Copyright 2010 - 2021 Hitachi Vantara.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.pentaho.mongo.wrapper;
+
+import com.mongodb.*;
```

Comment:
This import needs to be changed; we have to import all classes from a package separately, even if we end up importing the entire package.